File: model_helper.py
from PY import mcm;

mcm.init();
model, categories = mcm.load_model_uncompressed("model.umcm");
print("mcm.py\n Press !exit to exit");

while(True):
    text = input("Predict >>> ");
    if(text=="!exit"): exit(0);
    prediction = mcm.make_predict(model, categories, text);
    results = mcm.analyse_prediction(prediction, categories);
    print(results);

File: model_helper.js
var mcm = require("./JS/mcm.js");

(async () => {
    await mcm.init();
    let [training_resources, categories] = await mcm.create_resources_from_files("./source");
    let model = mcm.create_weights(training_resources, categories);

    await mcm.write_model(model, categories, "./model.mcm");
    //let [model, categories] = await mcm.load_model("./model.mcm");

    //let res = mcm.make_predict(model, categories, "it is licensed with gpl");
    //console.table([categories,[res[0]/(res[0]+res[1])*100+"%",res[1]/(res[0]+res[1])*100+"%"],res]);
})();